docs/install.rst

```diff
@@ -99,13 +99,14 @@ Python package dependencies
 - scikit-learn >=0.19.1
 - scipy >=1.1.0
 - seaborn >=0.9.0
+- joblib >=0.11
```

Comment:
I don't see this in the requirements.txt file? Should be consistent between the two.

Reply:
It's in the requirements.txt file for the PR #164, which I assumed would be merged first